stix_shifter_modules/nozomi/configuration/config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "connection": {
+    "type": {
+      "displayName": "Nozomi",
+      "group": "nozomi"
+    },
+    "host": {
+      "type": "text",
+      "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9_:/\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9_:/\\-]*[A-Za-z0-9])$"
+    },
+    "port": {
+      "type": "number",
+      "default": 443,
+      "min": 1,
+      "max": 65535
+    },
+    "help": {
+      "type": "link",
+      "default": "data-sources.html"
+    },
+    "selfSignedCert": {
+      "type": "password",
+      "optional": true
+    }
+  },
+  "configuration": {
+    "auth": {
+      "type": "fields",
+      "key_name": {
```

Comment:
Given that this is an API Key based application, the internal name here should be api_key_name and api_key.

Reply:
This would only be the internal key we use. The display should still show "Key Name" to match the UI, however internally, this is really just an API Key and the API Key Name in functionality.

Reply:
If we kept api_key_name and api_key_token, we would need to change this to key_name and key_token again while making an API call. 

Reply:
Ah, I see you just directly provide the auth as json here
`return await self.client.call_api(self.TOKEN_ENDPOINT, 'POST', headers=self.headers, data=json.dumps(self.auth),
`

in that case, it's fine to leave it as is.